praetorian_cli/ui/aegis/commands/job_helpers.py

```
@@ -335,6 +329,106 @@ def configure_parameters(menu, capability_info, has_credential=False):
+    """Create an asset for the given target and return (target_key, target_display).
+
+    Ensures the asset exists in the API before returning so that job execution
+    can find a matching target.
+    """
+    target_key = f"#asset#{hostname}#{target_input}"
+    target_display = f"asset {target_input}"
+
+    try:
+        menu.console.print(f"  [{colors['dim']}]Creating asset {target_input}...[/{colors['dim']}]")
+        menu.sdk.assets.add(group=hostname, identifier=target_input)
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)